### DISCUSSION TITLE
New subscription lightRASP

### DISCUSSION THREAD

**Discussion by syakymchuk** [talsec/Free-RASP-Community#8] [General]
Dear community, 

We are considering introducing the new low-cost subscription plan **lightRASP**.  The early-bird discounted pricing for a yearly subscription will look like this below (the first month is free) : 

- Up to 100K App downloads  $ 1,980 / year
- 100K to 1M App downloads  $ 4,800 / year
- 1M+ App downloads    $ 8,800 / year
NB: Early bird season is over  `\_(ツ)_/`

The primary purpose of the plan is to bridge the gap between freeRASP and Business RASP+ with [AppiCrypt SDKs](https://www.talsec.app/appicrypt?utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz--pd6tkPe8LApoeDTJEp9HfFNutdPVoYrBYXQbf7xv0B6IZDiabVcu3BbjI1vk23D6SoZYh). This lightRASP subscription targets the following medium-level app security goals: 
- Basic RASP to address OWASP Resilience compliance (Root/JB, Emulator, Frida detection, Installation Sources, ...)
- Code Tampering and App cloning protection 
- Configurable logs collection to setup own monitoring and control over security audit data 
- (coming soon) Obfuscating voluntary App's secrets (e.g. customers' API keys, hostnames, end-points, etc.)
- Commercial SLA    

Your comments and questions are very welcome!

[Special conditions](https://github.com/orgs/talsec/discussions/7) are available for SW houses as multi-app issuers, white-label app issuers, SDK vendors, and resellers.

 I am happy to answer any questions and provide further details. Don't hesitate to answer this thread or schedule a meeting in [Calendly here](https://calendly.com/sergiy-yakymchuk-1/60min?month=2023-01&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz--pd6tkPe8LApoeDTJEp9HfFNutdPVoYrBYXQbf7xv0B6IZDiabVcu3BbjI1vk23D6SoZYh).

Have a great day!

Kind regards, 
Sergiy Y.
Talsec co-founder

**Comment by ppeti** [comment]
has anything changed since February 2023? 
because I cannot find any information on the website.

best regards,

**Comment by syakymchuk** [comment]
Hello @ppeti ,

The lightRASP is available (usually offered to or via partners) but we can discuss. Contact me if you are interested to know more [Calendly here](https://calendly.com/sergiy-yakymchuk-1/).
